repose-aggregator/components/filters/translation/pom.xml

```diff
@@ -87,11 +86,6 @@
             <artifactId>xalan</artifactId>
             <version>2.7.1</version>
         </dependency>
-        <dependency>
```

Comment:
if it's using it directly, it keeps it's dependency. we do not count on transitive dependencies. ever.